yazi-config/src/theme/filetype.rs

```diff
@@ -33,6 +53,8 @@ impl Filetype {
 		struct FiletypeRule {
 			name: Option<Pattern>,
 			mime: Option<Pattern>,
+			#[serde(rename = "type")]
```

Comment:
The `type` field name is a proposal, it makes sense to me but we could use something else if it's better